rl_loop/train_and_validate.py

```diff
@@ -66,14 +74,27 @@ def train():
         print("Training failed!")
         return completed_process
 
+    # Train.py already copies the {data,index,meta} files to $BUCKET/models
     # Persist the checkpoint two ways:
     # Freeze the .ckpt file in the work_dir for the TPU selfplayers
     # Freeze a non-tpu version of the graph for later GPU use.
-    dual_net.freeze_latest_checkpoint_tpu()
-    dual_net.freeze_graph(save_file)
+    latest_checkpoint = tf.train.latest_checkpoint(fsdb.working_dir())
+    freeze(latest_checkpoint, use_tpu=True)
```

Comment:
change this kwarg name to `rewrite_tpu` to prevent confusion with the flagname?

Reply:
Done.